numpy/core/tests/test_multiarray.py

```diff
@@ -6453,6 +6453,19 @@ def test_relaxed_strides(self):
             shape, strides = get_buffer_info(arr, ['C_CONTIGUOUS'])
             assert_(strides[-1] == 8)
 
+    def test_out_of_order_fields(self):
+        dt = np.dtype(dict(
+            formats=['<i4', '<i4'],
+            names=['one', 'two'],
+            offsets=[4, 0],
+            itemsize=8
+        ))
```

Comment:
These would look better on the previous line. Python indentation isn't like C blocks.

Reply:
Used to writing things like this from javascript - for deeply nested parens, it makes things a lot clearer. Here you're right, it adds little value.